model_signing/manifest/manifest.py

```diff
@@ -14,14 +14,48 @@
 
 """Machinery for representing a serialized representation of an ML model.
 
-Currently, we only support a manifest that wraps around a digest. But, to
-support incremental updates and partial signature verification, we need a
-manifest that lists files and their digests. That will come in a future change,
-soon.
+When saving a model, or signing an existing one, we build a manifest from the
+model (either by using a `serialization.Serializer` or building it from helper
+methods that use precomputed values to reduce amount of I/O involved). This is
+the manifest used for generating a signature of the model.
+
+When testing the integrity of a model, we start with the model path and model
+type. From this, we extract the path to the signed manifest. Next step is to
+verify the authenticity of the signature, after which we extract an in-memory
+representation of the _expected_ manifest. To verify the integrity of the model,
+we run another `serialization.Serializer` instance to compute the _actual_
+manifest of the model as we're seeing it. If these two manifests agree then the
+model integrity is maintained.
+
+In the simplest case, we are working with `DigestManifest` objects. Here, the
+two manifests are conforming iff the digests are the same. A more complex case
```

Comment:
```suggestion
two manifests agree if and only if the digests are the same. A more complex case
```

(it looks like you otherwise changed 'conform' to 'agree')

Reply:
Thanks. Right, I missed this one